misc/python/materialize/checks/pg_cdc.py

```diff
@@ -174,27 +161,19 @@ def validate(self) -> Testdrive:
                 F 400 97350
                 G 300 97350
                 H 200 97350
+                # Confirm that the primary key information has been propagated from Pg
+                >[version>=5000] SELECT key FROM (SHOW INDEXES ON postgres_source_tableA);
```

Comment:
@philip-stoev This is failing in the random upgrade tests in v0.53.0: https://buildkite.com/materialize/nightlies/builds/2395#018829d4-ffe0-49eb-a3dc-a696700e5b40

You previously fixed it like this: https://github.com/MaterializeInc/materialize/pull/18549 But this check is still in there, but with slightly different timing now because we have a `select mz_version_num()` before it.

Do you have any idea on how to fix it better?

Reply:
0.50.0-dev also returns 5000 in `mz_version_num()`

Reply:
So I guess this check needs to remain on the python side

Reply:
Sure, done, that fixes the red nightly, but to be honest I'm lacking the context of this issue. Should the base version matter for the result here? If so, then this is fine.

Reply:
Yes, the base version matters, as this is when the list of columns that participate in the index are determined. Different versions create indexes with a different set of columns. You have a missing import though:

```
[2023-05-19T15:15:53Z] NameError: name 'MzVersion' is not defined
```

Reply:
I see, thanks